content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -232,18 +233,37 @@ edited to correct any typos, or modified to remove any proprietary or client inf
 users to locate during a search. When all adjustments have been made, click :guilabel:`Create and
 View Post`.
 
+Create a Helpdesk ticket from a forum post
+------------------------------------------
+
+Forum posts submitted by portal users can be converted to **Helpdesk** tickets.
+
+To create a ticket, navigate to a forum post, and click the :icon:`fa-ellipsis-h`
+:guilabel:`(ellipsis)` icon. Then, click :guilabel:`Create Ticket`.
+
+.. image:: help_center/help-center-create-ticket.png
+   :alt: A forum post with the create ticket option visible.
+
+This opens a :guilabel:`Create Ticket` pop-up. Make any necessary edits to the :guilabel:`Create
+Ticket` field. Then, confirm the :guilabel:`Helpdesk Team` the ticket should be assigned to.
+
+Click :guilabel:`Create & View Ticket` or :guilabel:`Create Ticket`.
+
+.. note::
+   The original forum post is linked in the *chatter* on the new ticket.
```

Comment:
No formatting is necessary for chatter :)

p.s. Im sorry to hear that Joel's dining chair cracked :(
```suggestion
   The original forum post is linked in the chatter on the new ticket.
```